### ISSUE TITLE
IE11 svg output bug

### ISSUE THREAD

**Issue by chitgoks** [szimek/signature_pad#220]
IE11 does not output any paths in the generated svg.

See sample output.
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 430 150">undefined</svg>

no error message in IE 11 console.

off topic: could you possibly provide a trim feature for svg?

**Comment by szimek** [comment]
Turns out that IE doesn't support `innerHTML` property on SVG elements. Because why should it :/

Anyway, I found a workaround:
```js
  let body = svg.innerHTML;

  // IE hack for missing innerHTML property on SVG elements
  if (body === undefined) {
    const paths = svg.childNodes;
    const serializer = new XMLSerializer();
    body = '';
    for (let i = 0; i < paths.length; i += 1) {
      body += serializer.serializeToString(paths[i]);
    }
  }
```
I tested it only in IE11, but `XMLSerializer` is supported since IE9. I'll publish a new beta version with this workaround later today.

**Comment by chitgoks** [comment]
great news! thanks!

**Comment by szimek** [comment]
Closed by ff3dffffc9ba5149f45a5d6bbdbe85bc975db77d.

**Comment by chitgoks** [comment]
hi szimek. what happened to the signature_pad.min.js? it's not there anymore. thanks

**Comment by szimek** [comment]
I removed built files from the repo. They're available in npm package or cdnjs (though it looks like only stable builds are available there).

**Comment by chitgoks** [comment]
yes. you are right. npm does not contain the latest, also not minimized. thanks

**Comment by chitgoks** [comment]
hi @szimek. the output svg has a problem. all the path tags have an xmlns attribute 

e.g.
<path xmlns="http;//www.w3.org/2000/svg" 

only the first path tag should have it.

**Comment by szimek** [comment]
@chitgoks 
- npm package does contain all 3 built files. If you install it with `npm install signature_pad@beta` you have the minified version in `./node_modules/signature_pad/dist/signature_pad.min.js`

- no they don't. If you run something like this on the demo page:
```js
atob(signaturePad.toDataURL('image/svg+xml').split(',')[1])
```
you should get output like this:
```
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 1300 1076"><path d="M 170.000,213.500 C 170.000,214.500 170.000,214.500 170.000,215.500" stroke-width="5.519" stroke="black" fill="none" stroke-linecap="round"></path><path d="M 170.000,215.500 C 170.000,219.500 170.000,219.500 170.000,223.500" stroke-width="3.115" stroke="black" fill="none" stroke-linecap="round"></path><path d="M 170.000,223.500 C 170.000,224.000 170.000,224.000 170.000,224.500" stroke-width="4.263" stroke="black" fill="none" stroke-linecap="round"></path><path d="M 170.000,224.500 C 170.000,225.500 170.000...
```
I tried it in Chrome and FF.

**Comment by chitgoks** [comment]
hi. sorry i was not clear. yes, only in IE does it output all path tags with xmnls attribute. 

it's ok i am in no rush. i will wait.

**Comment by szimek** [comment]
@chitgoks Can you check out the version at http://szimek.github.io/signature_pad ? It still has `xmlns` attribute for `<circle>` elements (no idea why), but doesn't seem to have it for `<path>` elements anymore.

**Comment by chitgoks** [comment]
@szimek: i am not sure when <circle> will appear. i have not experienced that.

But as for <path>. it is now visible in IE.

Thanks. Hope there will be a min version as well.

**Comment by szimek** [comment]
@chitgoks `<circle>` appears in the generated SVG when you make a single dot. I'll release it as beta 8 (though I still need to fix #219 and #221 before the final release), as soon as I can confirm both `<path>` and `<circle>` show up properly.

**Comment by szimek** [comment]
@chitgoks Hey, I've just released 1.6.0-beta.8, which should fix it. Could you check it out?

**Comment by chitgoks** [comment]
hi @szimek  Confirmed working.
